src/main/java/mate/academy/model/User.java

```diff
@@ -53,8 +57,25 @@ public void setSalt(byte[] salt) {
     @Override
     public String toString() {
         return "User{"
-            + "id=" + id
-            + ", email='" + email + '\''
-            + '}';
+                + "id=" + id
+                + ", email='"
+                + email + '\''
+                + '}';
+    }
```

Comment:
Consider excluding the `password` and `salt` fields from the `toString` method to avoid exposing sensitive information.